xhtml2pdf/reportlab_paragraph.py

```diff
@@ -460,6 +460,11 @@ def _getFragWords(frags):
     hangingStrip = False
     for f in frags:
         text = f.text
+        if type(text) is bytes:
+            try:
+                text = text.decode('utf-8')
+            except Exception:
```

Comment:
As #290 expresses, this is something we want to avoid for the future. I suggest you just fail hard. Nobody should live with silent decode errors IMO.